contrib/inventory/azure_rm.py

```diff
@@ -299,9 +316,9 @@ def __init__(self, args):
         self.log("setting subscription_id")
         self.subscription_id = self.credentials['subscription_id']
 
-        if self.credentials.get('client_id') is not None and \
-           self.credentials.get('secret') is not None and \
-           self.credentials.get('tenant') is not None:
+        if self.credentials.get('credentials'):
+            self.azure_credentials = self.credentials.get('credentials')
```

Comment:
i think i would do:

self.azure_credentials = self.credentials.get('credentials')

if not self.azure_credentials:
    ....

Reply:
It has 4 conditional check here, I would like to choose the shortest one. 😃